packages/common/src/TimeUtils.js

```diff
@@ -14,6 +16,16 @@ const averageBlockTimeSeconds = async (/* lookbackSeconds */) => {
   }
 };
 
+// Sets fromBlock to the value of an environment variable if one is set. This can be set to 0 to make tests work with Ganache, or any other value needed for a production script or bot.
+function getFromBlock() {
+  if (process.env.FROM_BLOCK) {
+    return process.env.FROM_BLOCK;
```

Comment:
This could be a problem since env vars are parsed as strings and someone is expecting a number. Also if this is set to "0" is it considered truthy?

Reply:
In earlier testing, when setting the `FROM_BLOCK` to zero in an environment variable this worked out okay. Think I should add an explicit conversion to a number, though?

Reply:
yea i think probably cast to number on return